src/Web/Client/Pages/Employees.razor

```diff
@@ -1,18 +1,99 @@
+        <MudStack Row=true Class="d-flex justify-space-between mb-5">
+            <MudText Typo="Typo.h4">Employee</MudText>  
+            <MudButton ButtonType=ButtonType.Submit Variant="Variant.Filled" StartIcon="@Icons.Material.Filled.Save" Color="Color.Tertiary" Size="Size.Medium">Save</MudButton>
+        </MudStack>
+        <MudDivider />
+        <MudStack Class="mt-3 pa-4" Style=" max-width: 500px">
+                <MudTextField @bind-Value=_createEmployee.Email Label="Email" InputType="InputType.Email" Margin="Margin.Dense" Variant="Variant.Outlined" For="@(() => _createEmployee.Email)"/>
+                <MudStack Row=true>
+                    <MudTextField @bind-Value=_createEmployee.FirstName Label="First name" Margin="Margin.Dense" Variant="Variant.Outlined" For="@(() => _createEmployee.FirstName)" />
+                    <MudTextField @bind-Value=_createEmployee.LastName Label="Last name" Margin="Margin.Dense" Variant="Variant.Outlined" For="@(() => _createEmployee.LastName)" />
+                </MudStack>
+                <MudStack Row=true>
+                    <MudTextField @bind-Value=_createEmployee.Username Label="Username" Margin="Margin.Dense" Variant="Variant.Outlined" For="@(() => _createEmployee.Username)" />
+                    <MudTextField @bind-Value=_createEmployee.PhoneNumber Label="Phone" Margin="Margin.Dense" Variant="Variant.Outlined" />
+                </MudStack>
+                <MudStack Row=true>
+                    <MudTextField @bind-Value=_createEmployee.Password Label="Password" InputType="InputType.Password" Margin="Margin.Dense" Variant="Variant.Outlined" For=@(() => _createEmployee.Password) />
+                    <MudTextField @bind-Value=_createEmployee.ConfirmPassword Label="Confirm password" InputType="InputType.Password" Margin="Margin.Dense" Variant="Variant.Outlined" For="@(() => _createEmployee.ConfirmPassword)" />
+                </MudStack> 
+        </MudStack>
+    </EditForm>
+}
+
 
-<PageTitle>Employees</PageTitle>
 
-<h1>Employees</h1>
 
-<p role="status">Current count: @currentCount</p>
+<SectionContent SectionName="@nameof(MenuItemList)">
+    <MenuItemList Service="EmployeesService" BaseRoute="/employees" />
+</SectionContent>
 
-<button class="btn btn-primary" @onclick="IncrementCount">Click me</button>
 
 @code {
-    private int currentCount = 0;
+    [Parameter] public string? Id { get; set; }
 
-    private void IncrementCount()
+    GetEmployee _getEmployee = null!;
+    CreateEmployeeRequest _createEmployee = new CreateEmployeeRequest();
+    bool _loading = false;
+
+    protected override async Task OnParametersSetAsync()
+    {
+        _loading = true;
+        if (Id is not null)
+        {
+            _getEmployee = await EmployeesService.GetEmployeeById(Id);
+        }
+        _loading = false;
+        StateHasChanged();
+    }
+
+    private async void CreateOrUpdate()
     {
-        currentCount++;
+        if (Id is not null)
```

Comment:
if (Id is null) {
   create employee
}

Reply:
Changed